pkg/apis/networking/types.go

```diff
@@ -361,6 +360,15 @@ type IngressRule struct {
 	// Incoming requests are matched against the host before the
 	// IngressRuleValue. If the host is unspecified, the Ingress routes all
 	// traffic based on the specified IngressRuleValue.
+	//
+	// Host can be "precise" which is an domain name without the terminating dot of a network host (e.g. "foo.bar.com")
+	// or "wildcard", which is a domain name prefixed with a single wildcard label (e.g. "*.foo.com").
+	// The wildcard character '*' must appear by itself as the first DNS label and matches only a single label.
+	// You cannot have a wildcard label by itself (e.g. Host == "*").
+	// Requests will be matched against the Host field in the following way:
+	// 1. If Host is precise, the request matches this rule if the http host header is equal to Host.
+	// 2. Host is a wildcard, then the request matches this rule if the http host header is to equal to the
```

Comment:
I think 1 and 2 should either both start with "If" or not, but I'd lean towards both starting with "If".